cobalt/build/gn.py

```diff
@@ -84,10 +84,10 @@ def write_build_args(build_args_path, original_lines, dict_settings,
         f'The following args cannot be set in configs: {controlled_args}')
   gen_comment = '# Set by gn.py'
   with open(build_args_path, 'w', encoding='utf-8') as f:
-    if os.getenv('USE_RBE', default='0') == '0':
-      f.write(f'cc_wrapper = "{CC_WRAPPER}" {gen_comment}\n')
-    else:
+    if os.getenv('IS_CI', default='0') == '0':
```

Comment:
While this isn't really a negative conditional, it kind of feels like one; I have a slight preference for `if os.getenv('IS_CI', default='0') == '1'` (which to me does what it reads like at a glance, i.e. `if IS_CI`)

Reply:
changed conditional as requested.